src/components/renderers/renderer-service.ts

```diff
@@ -29,10 +29,14 @@ class RendererServiceImpl implements RendererService {
         this.renderer.push({directiveName: directiveName, tester: tester});
     }
     getBestComponent(element: IUISchemaElement, dataSchema: any, dataObject: any): string {
-        let bestRenderer = _.maxBy(this.renderer, renderer =>
-            renderer.tester(element, dataSchema, dataObject, this.pathResolver)
-        );
-
+        let bestRenderer = _.maxBy(this.renderer, renderer => {
+            let result = renderer.tester(element, dataSchema, dataObject, this.pathResolver);
+            return result === NOT_FITTING ? null : result;
```

Comment:
Why do we need to return `null`? Why don't we compare `bestRenderer` with `NOT_FITTING`?